rabbitmq/manifest.json

```diff
@@ -11,6 +11,6 @@
     "mac_os",
     "windows"
   ],
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
I think we would want to have this be a bugfix update, so I believe the version number should be `1.3.1`